packages/pages/src/components/image/types.ts

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+
+export type ThumbnailType = {
+  height: number;
+  width: number;
+  url: string;
+};
+
+export type ImageType = {
+  image: {
+    height: number;
+    width: number;
+    url: string;
+    thumbnails?: ThumbnailType[];
+  };
+};
+
+export const ImageLayout = {
```

Comment:
this and the type have the same name which isn't ideal -- maybe this can be called ImageLayoutOption?